### PR TITLE
Feat/Add object notification

### PR DESCRIPTION
**Pull request by carpawell**
See https://github.com/nspcc-dev/neofs-node/pull/1190 for testing steps.

### CODE REVIEW

File: services/storage/docker-compose.yml [hunk 1]
```diff
@@ -52,6 +55,9 @@ services:
       - ./../../vendor/neofs-cli:/neofs-cli
       - ./healthcheck.sh:/healthcheck.sh
       - ./s04tls.crt:/etc/ssl/certs/s04tls.crt
+      - ../nats/client-cert.pem:/nats.tls.cert
```
Comment (realloc) [inline services/storage/docker-compose.yml:58]: I'd prefer not to use / for anything. Let's create a separate directory, preferably using same path as normal node setup would do.

**Comment by realloc** [comment]
Instead of merging to `master` I'd suggest using a separate branch until full feature set release.

**Comment by carpawell** [comment]
@realloc, sure

if fact, it is really *draft*, not expected anybody looking at it yet, but if we already started, I have some questions: do we need any trivial subscriber in `dev-env`? just smth that could read notifications from the nodes and log them, or there are only gateways that are going to subscribe for messages?

also, do we need all 4 nodes producing messages?

**Comment by realloc** [comment]
> do we need any trivial subscriber in dev-env?

It looks like for debug purposes NATS and S3 GW logging would be enough. No need for a new service.

> do we need all 4 nodes producing messages?

Yes. Otherwise, we can miss ticks for `REP 1` containers.

**Comment by carpawell** [comment]
The notifications feature has been tested and already merged in the `master`. I have tested that PR one more time and changed docker volumes (to be not `/`).

**Review by realloc — APPROVED** [review]

**Review by KirillovDenis — APPROVED** [review]

**Pull request merged**